composer.json

```diff
@@ -6,7 +6,8 @@
     },
```

Comment:
Přidal bych požadavek i na minimální verzi PHP tak aby to nedělalo problém ve starých aplikacích.

Reply:
dát tam 7.1?

Reply:
jj